edb/schema/indexes.py

```diff
@@ -125,6 +129,23 @@ class Index(
         inheritable=False,
     )
 
+    params = so.SchemaField(
+        s_func.FuncParameterList,
+        coerce=True,
+        compcoef=0.4,
+        default=so.DEFAULT_CONSTRUCTOR,
+        inheritable=True,
+    )
+
+    kwargs = so.SchemaField(
+        s_expr.ExpressionDict,
+        coerce=True,
+        compcoef=0.4,
```

Comment:
Does `ALTER` work for any of this yet? If not maybe compcoef should be 0?

Reply:
Set to 0